### DISCUSSION TITLE
Google Fonts on Classic Theme

### DISCUSSION THREAD

**Discussion by NimbleOnion** [PrestaShop/PrestaShop#34808] [Q&A] [answered]
Hi, it's my first time using prestashop and I want to change the default font to the Roboto font. I already added the HTML code in head.tpl, but i wonder what do I have to change in the theme.css file in order to set that font as the displayed one?
I'm using PS 8.1.1 and the Classic theme 2.1.1

**Comment by kpodemski** [comment, marked as answer]
Hello @NimbleOnion 

>  but i wonder what do I have to change in the theme.css file in order to set that font as the displayed one?

I'm afraid it doesn't work like that. You have to modify the source files to change the fonts. You can find more information on how to work with themes in the developer documentation:
https://devdocs.prestashop-project.org/8/themes/

You'd have to download the sources:
https://github.com/PrestaShop/classic-theme/archive/refs/tags/2.1.2.zip

Edit files inside `_dev`, compile them using Webpack, and then use the compiled theme.css file.
